src/shell/subproc.zig

```diff
@@ -955,7 +975,7 @@ pub const PipeReader = struct {
     state: union(enum) {
         pending: void,
         done: []u8,
-        err: bun.sys.Error,
+        err: ?JSC.SystemError,
```

Comment:
If you want to keep in the bun family `bun.sys.SystemError` is equiv to `JSC.SystemError`.